packages/expo-camera/src/CameraView.tsx

```diff
@@ -227,6 +240,13 @@ export default class CameraView extends Component<CameraViewProps> {
     return this._cameraRef.current?.record(recordingOptions);
   }
 
+  /**
+   * Pauses or resumes the video recording. Only has an effect if there is an active recording. On `iOS`, this method only supported on `iOS` 18.
```

Comment:
nit: maybe add an example to use `getSupportedFeatures` for feature detection in the doc comment as well as generated doc